schema/workflow.yaml

```
@@ -1292,6 +1292,12 @@ $defs:
           - title: ExpressionDataSchema
             $ref: '#/$defs/runtimeExpression'
             description: An expression based event data schema.
+      data:
+        title: EventData
+        description: The event's payload data
+        oneOf:
+          - $ref: '#/$defs/runtimeExpression'
+          - type: object 
```

We must add the following property, or else it won't work on some validators:

```suggestion
          - type: object 
            additionalProperties: true
```

---

I think this comments overlaps with mathias one, lets dicuss it there